homeassistant/const.py

```diff
@@ -2,7 +2,7 @@
 """Constants used by Home Assistant components."""
 MAJOR_VERSION = 0
 MINOR_VERSION = 46
-PATCH_VERSION = '0.dev0'
+PATCH_VERSION = '0.dev1'
```

Comment:
Change was done so that people running dev would also get the update. Otherwise we would break anyone running dev. This prevents thats.